book/chapters/08-hw/contents.tex

```diff
@@ -976,7 +972,7 @@ \subsection{Virtualizarea în hardware}
 mai multe sisteme de operare în același timp s-a introdus conceptul de
 \textbf{virtualizare} (\textit{virtualization}): dacă se rulează mai multe sisteme de
 operare în același timp, fiecare din ele va vedea o instanță virtuală a fiecărei
-componente a sistemului (CPU virtual, memorie virtuală, disc virtual etc). Se
+componente a sistemului (CPU virtual, memorie virtuală, disc virtual, etc.). Se
```

Comment:
```suggestion
componente a sistemului (CPU virtual, memorie virtuală, disc virtual etc.). Se
```

Reply:
https://bucatidincarti.wordpress.com/2009/07/02/greseli-de-punctuatie-in-folosirea-lui-etc/